### PR TITLE
Karatnyk

### PR DESCRIPTION
**Pull request by KatyaKaratnyk**
add WaysToSumUsingArray
add PathWithoutCrossing

### CODE REVIEW

**Review by mpaselskyi — APPROVED** [review]

File: src/main/java/com/softserve/algorithms/WaysToSumUsingArray.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.softserve.algorithms;
+
+public class WaysToSumUsingArray {
+	
+	public int waysToSumToNUsingArrayElementsWithRepetition(int array[], int n) {
```
Comment (tania-klueva) [inline src/main/java/com/softserve/algorithms/WaysToSumUsingArray.java]: Methods name should be verbs

File: src/main/java/com/softserve/algorithms/PathWithoutCrossing.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package com.softserve.algorithms;
+
+public class PathWithoutCrossing {
+	
+	public int pathsWithoutCrossing (int numberPoints) {
```
Comment (tania-klueva) [inline src/main/java/com/softserve/algorithms/PathWithoutCrossing.java]: Methods name should be verbs

**Review by tania-klueva — CHANGES_REQUESTED** [review]
Methods name should be verbs

**Review by tania-klueva — CHANGES_REQUESTED** [review]
still wrong names

**Review by tania-klueva — APPROVED** [review]

**Pull request merged**